Forwarded-Upstream: not yet
Author: Raphael Hertzog <hertzog@debian.org>
Comment:
 Fix a lintian I: message about improper usage of minus instead
 of hyphen.

--- a/docs/man/django-admin.1
+++ b/docs/man/django-admin.1
@@ -183,7 +183,7 @@ The domain of the message files (default
 .TP
 .I \-e, \-\-extension=EXTENSION
 The file extension(s) to examine (default: ".html", separate multiple
-extensions with commas, or use -e multiple times).
+extensions with commas, or use \-e multiple times).
 .TP
 .I \-s, \-\-symlinks
 Follows symlinks to directories when examining source code and templates for
